endpoints/info/bidders.go

```diff
@@ -49,12 +57,33 @@ func NewBidderDetailsEndpoint(infos adapters.BidderInfos, aliases map[string]str
 
 	// Add in any default aliases
 	for aliasName, bidderName := range aliases {
+		// add the alias bidder info into "allBidderInfo" map
+		aliasInfo := infos[bidderName]
+		aliasInfo.AliasOf = bidderName
+		allBidderInfo[aliasName] = aliasInfo
+
+		// JSON encode core bidder info for the alias and add it to the "responses" map
 		responses[aliasName] = createAliasInfo(responses, aliasName, bidderName)
 	}
 
+	allBidderResponse, err := json.Marshal(allBidderInfo)
+	if err != nil {
+		glog.Fatal("Failed to JSON-marshal all bidder info data.")
+	}
+
 	// Return an endpoint which writes the responses from memory.
 	return httprouter.Handle(func(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
 		forBidder := ps.ByName("bidderName")
+
+		// If the requested path was /info/bidders/all then return info about all the bidders
+		if forBidder == "all" {
```

Comment:
might be simpler to have `responses["all"] = allBidderResponse` above and skip this if condition?

Reply:
Yeah, that's a good idea. Updated the PR. Thanks @hhhjort!